releasenotes/notes/fix_to_dict_decimals-2f0d3bb865fb09a9.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    The method :meth:`qiskit.quantum_info.Statevector.to_dict` has a `decimals` keyword to round the results, which was not effective because of a typo. This fix resolves the apparent typo located in :method:`QuantumState._vector_to_dict()`. 
```

```suggestion
    The method :meth:`.Statevector.to_dict` will now respect its ``decimals`` keyword argument, which it previously ignored.
```

A minor tweak: rST syntax needs two backticks around code (but not cross-references), so it's ` ``decimals`` `, and then just a text tweak because the release note doesn't appear "attached" to the PR or anything, so it needs to be understandable without things like "this fix", and the readers don't necessarily need to know what private function the problem was in.